packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarBackButtonIOS.ios.js

```diff
@@ -40,6 +40,7 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     zIndex: 2,
     alignItems: 'center',
+    height: '100%',
```

Comment:
Doesn't this need `justifyContent: 'center'` too?

Reply:
Nope, it's a row and already has `aligntItems: 'center'`